### DISCUSSION TITLE
Please fix Rain keyboard lighting option!

### DISCUSSION THREAD

**Discussion by fluent2** [seerge/g-helper#2291] [Ideas]
Hi! Okay so I use this mode exclusively with ASUS software but it's broken in G-Helper. Basically, it needs an option to use TWO colors, not just one. Color choice doesn't work at all with this mode to begin with, it just defaults to red and white for some reason, but then to fix it it would need two options, i.e. Color 1 and Color 2, preferably able to set 2 custom colors. Hope you'll throw this fix in sometime real quick since it shouldn't be too hard to add and Rain is probably a pretty popular lighting option that people would love to use with your tool. Awesome tool by the way, thanks for it! - flu

**Comment by seerge** [comment]
@fluent2 hello, this was answered here multiple times. 

G-Helper shows modes supported by keyboard on hardware level. If color is not available, it means this mode can't be customized with other colors (good example is Rainbow).

**Comment by fluent2** [comment]
Hi. I did do a search before posting and didn't find anything. That said, Rain preset is customizable with Armoury Crate, though. Does that mean that the only way to customize it would be to use AC again? Is it possible to use AC just for that purpose and use G-Helper for all the other settings?

**Reply by seerge** [reply]
AC can run software emulation of some modes, which means it sends a color commands few times a second, imitating color change. It's very resource heavy process and against G-Helper's idea to be lightweight.

**Reply by seerge** [reply]
P.S. If you want to try more customization for RGB you can also use https://openrgb.org/
